src/components/molecules/OakModal/OakModal.tsx

```diff
@@ -79,7 +80,7 @@ const logoSrc = `https://${process.env.NEXT_PUBLIC_OAK_ASSETS_HOST}/${process.en
 export const OakModal = ({
   children,
   footerSlot,
-  domContainer = document.body,
```

Comment:
This was the source of the error I was encountering locally — until now we hadn't tried the modal under SSR so this wasn't a problem. On the server `document` isn't defined so things break when this component is rendered.